ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -29,6 +29,10 @@ if(_source_files)
 
   # Get include paths for added targets
   set(_all_include_dirs "")
+  if(DEFINED AMENT_CMAKE_CPPCHECK_INCLUDE_DIRS)
```

Comment:
I would suggest to use lowercase for the `AMENT_CMAKE_CPPCHECK` to match the package name and other variables in style.

Reply:
now that I think about, maybe a completely different name could be good. Might be confusing with the default c++ like include dirs.
what about `ament_cmake_cppcheck_EXTERNAL_DIRS`?

Reply:
`ament_cmake_cppcheck_EXTERNAL_INCLUDE_DIRS`?